src/components/DrinkPage.js

```diff
@@ -1,10 +1,8 @@
 import React, {useState} from "react";
 import {useSelector} from "react-redux";
-
-import FilterDrink from "./FilterDrink";
-import DrinkCard from "./UI/DrinkCard";
-
 import {useEffect} from "react";
+import FilterDrink from "./FilterDrink";
+import {DrinkCard} from "./UI/index";
 import {useActions} from "../helpers/hooks/useActions";
 import {fetchDrink} from "../redux/actions/drinkActions";
 
```

Comment:
You don't use `props`.

small refactoring at 23 line of code 

```
const renderCard = () => {
        return filteredDrinkData.map(item => (
            <DrinkCard 
                key={item.id}
                img={item.img}
                title={item.title}
                size={item.size}
                price={item.price}
            />
        ));
    };
```

54 line of code - `;`